packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginCardWithValidation.js

```diff
@@ -204,7 +214,13 @@ class LoginCardWithValidation extends React.Component {
     });
   };
 
-  isUserNameValid = () => {
+  isEmailValid = () => {
```

Comment:
I think the method should remain as `isUserNameValid`, if follow-up PRs add more validation cases there would be no need to re-rename this method.